core/src/main/java/io/substrait/expression/proto/ProtoExpressionConverter.java

```diff
@@ -393,10 +397,7 @@ public Expression.SortField fromSortField(SortField s) {
         .build();
   }
 
-  public FunctionOption fromFunctionOption(io.substrait.proto.FunctionOption o) {
-    return ImmutableFunctionOption.builder()
-        .name(o.getName())
-        .addAllValues(o.getPreferenceList())
-        .build();
+  public static FunctionOption fromFunctionOption(io.substrait.proto.FunctionOption o) {
```

Comment:
It is somewhat awkward to make this static so it can be re-used in the `ProtoAggregateFunctionConverter`, but I also can't think of a better place to put this.